src/components/standalone/update/UploadImageDrawer.vue

```diff
@@ -119,7 +112,7 @@ async function uploadImageAndReboot() {
       <div>
         <NeFileInput
           :label="`${t('standalone.update.image_file')} (*.img.gz)`"
-          @change="handleFileUpload"
+          @select="handleFileUpload"
           :invalid-message="fileInputValidationError"
```

Comment:
I've just noticed this component doesn't have the `dropzoneLabel`, which should be required. We should probably add it (and now that i think about it, maybe it would be better to use a single shared `upload_file_description` translation entry, since we already have two identical dropzone labels for the other `NeFileInput`s used in the ui